rosetta/retriever/retriever_test.go

```diff
@@ -869,7 +873,114 @@ func TestRetriever_Transaction(t *testing.T) {
 
 		ret := retriever.BaselineRetriever(t, retriever.WithConverter(convert))
 
-		_, err := ret.Transaction(rosBlockID, mocks.GenericTransactionQualifier(0))
+		_, err := ret.Transaction(mocks.GenericRosBlockID, mocks.GenericTransactionQualifier(0))
+
+		assert.Error(t, err)
+	})
+}
+
+func TestRetriever_Sequence(t *testing.T) {
+	t.Run("nominal case", func(t *testing.T) {
+		t.Parallel()
+
+		validator := mocks.BaselineValidator(t)
+		validator.BlockFunc = func(blockID identifier.Block) (uint64, flow.Identifier, error) {
+			assert.Equal(t, mocks.GenericRosBlockID, blockID)
+
```

Comment:
Why do you put these spaces? Was wondering before. Seem superfluous to me.

Reply:
Return statement should not be cuddled with assertions IMO. The only case in which return statements can be cuddled, according to most linters and my previous code style guide, is when it's a function with only a return statement, or a definition followed by a return statement that returns what was defined.

Since it's a pretty common rule, I try to apply it consistently.